libexec/bats

```diff
@@ -55,7 +60,7 @@ expand_path() {
 BATS_LIBEXEC="$(abs_dirname "$0")"
 export BATS_PREFIX="$(abs_dirname "$BATS_LIBEXEC")"
 export BATS_CWD="$(abs_dirname .)"
-export PATH="$BATS_LIBEXEC:$PATH"
+export PATH="$BATS_LIBEXEC:$(getconf PATH 2>/dev/null):$PATH"
```

Comment:
You need the getconf'd paths to be before the standard system paths to ensure that the POSIX compatible utilities are used in preference to the standard system tools provided with Solaris.

As for why they're not there by default, you'd have to ask Oracle - they still ship their ancient tools (presumably for compatibility) as the default in preference to the slightly-less-broken POSIX tools.